resources/disksetup.rb

```diff
@@ -0,0 +1,34 @@
+# Resource properties
+property :disks, Array, required: true
+property :stripe_width, Integer
+property :tmp_disk_file, String, default: '/tmp/disks.txt'
+property :opts, Array
+
+load_current_value do |_new_resource|
+  configured_disks = ::Mapr::DiskSetup.configured_disks()
+  if configured_disks
+    disks configured_disks
+  else
+    current_value_does_not_exist!
```

Comment:
`current_value_does_not_exist! unless configured_disks`? rather than an if/else?

Reply:
rubocop prefer if/else ;)

And I will complete the code adding more checks here.